FirebasePerformance/Tests/Unit/Configurations/FPRRemoteConfigFlagsTest.m

```diff
@@ -65,7 +67,7 @@ - (void)testCacheResetAfterEverySuccessfulFetch {
                                 forKey:@"fpr_vc_session_sampling_rate"];
 
   // Trigger the RC config fetch
-  remoteConfig.fetchStatus = FIRRemoteConfigFetchStatusSuccess;
+  remoteConfig.fetchStatus = FIRRemoteConfigFetchAndActivateStatusSuccessUsingPreFetchedData;
```

Comment:
just double checking this diff was intentional and this shouldn't remain `FIRRemoteConfigFetchStatusSuccess`

Reply:
Yes - this fixes inconsistent enum type usage. More analysis may be needed, but this is a step in the right direction and the tests still pass.